documents/policies/Data Protection.md

```diff
@@ -0,0 +1,26 @@
+# WCA Data Protection Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to create an Advisory Committee with the responsibility and duty of researching, creating, implementing, managing, and enforcing all WCA Data Protection policies, contracts with external Data Processors of WCA Data, and managing tasks relating to Data Protection.
+
+## Policy
+The WCA Board hereby creates the WCA Data Protection Committee (WDPC) as an Advisory Committee of the WCA.
+
+1. The WCA Data Protection Committee is responsible for all matters relating to Data Protection.
+2. The WCA Data Protection Committee has the following rights and duties:
+   1. Creation, implementation, and enforcement of all WCA Data Protection policies:
+      1. Researching regulations globally to ensure the WCA adopts the policy requirements to satisfy all global regulations.
+      2. Engaging with legal council on all matters relating to Data Protection with WCA Board approval.
+      3. Understanding the scope of which Data Protection policies are required and duly advising the WCA Board.
+   2. Acting as WCA liaison for Data Protection matters:
+      1. Managing requests from Registered Speedcubers and WCA Staff on Data Protection
+         1. Managing requests relating to data subject’s right to be forgotten.
+            1. Liaison with the WRT to request data subject data be removed or anonymized within the WCA Database.
+            2. Liaison with the WST to request that data subject data be removed or anonymized within the WCA Website.
```

Comment:
nit "data subject data" might technically be the right term here, but it reads weird. I don't have a suggestion for a better phrase, sorry.

I'm not sure I understand the difference between "WCA Database" and "WCA Website" here. With very few hacky exceptions (like when we hardcoded the list of competitors with "In Memoriams"), ever bit of information like this *should* be in the WCA Database.

Long term, my vision would be for the WRT to handle all the removal/anonymization of data, but they would do it with tools written by the WST. In practice, right now I don't think it makes sense for the WST to invest a lot of time into building such tooling, when it's more practical to just go to the database and remove/scramble stuff by hand.

Reply:
How about "data subject's personal data" instead?

An example of something that was in the WCA website was a request to remove data from the old forums.

Reply:
"data subject's personal data" sounds good to me.

> An example of something that was in the WCA website was a request to remove data from the old forums.

Forum data is in our database.

Reply:
I agree with the removal of line 22. Thanks Jeremy for the feedback!

Reply:
I too concur. Thanks for the feedback.

Reply:
Change committed!